polyply/data/martini3/modifications.ff

```diff
@@ -35,3 +35,72 @@ BB {"replace": {"atype": "P1", "charge": 0.0}}
 NCAP-ter
 [ atoms ]
 BB {"replace": {"atype": "P2", "charge": 0.0}}
+
+[ modification ]
+GLU-HE1
+[ atoms ]
+SC1 {"replace": {"atype": "P2", "charge": 0}}
+
+[ modification ]
+GLU-HE2
+[ atoms ]
+SC1 {"replace": {"atype": "P2", "charge": 0}}
+
+[ modification ]
+ASP-HD1
+[ atoms ]
+SC1 {"replace": {"atype": "P2", "charge": 0}}
+
+[ modification ]
+ASP-HD2
+[ atoms ]
+SC1 {"replace": {"atype": "P2", "charge": 0}}
+
+[ modification ]
+LYS-LSN
+[ atoms ]
+SC2 {"resname": "LYS", "replace": {"atype": "SN6d", "charge": 0}}
+
+[ modification ]
+LYS-HZ3
+[ atoms ]
+SC2 {"replace": {"resname": "LYS"}}
+
+[ modification ]
+HIS-HP
+[ atoms ]
+BB  {"resname": "HIS", "replace": {"resname": "HIS"}}
+SC1 {"resname": "HIS", "replace": {"resname": "HIS"}}
+SC2 {"resname": "HIS", "replace": {"atype": "TP1dq", "charge": "0.5", "resname": "HIS"}}
+SC3 {"resname": "HIS", "replace": {"atype": "TP1dq", "charge": "0.5", "resname": "HIS"}}
+;[ edges ]
+;BB SC1
+;SC1 SC2
+;SC1 SC3
+;SC2 SC3
```

Comment:
are these commented out on purpose?

Reply:
yes. For some reason modifications.ff can't be read if they're included, and excluding them doesn't seem to affect them being applied correctly :man_shrugging: